src/status_im/ui/screens/browser/views.cljs

```diff
@@ -64,9 +65,15 @@
     [components/activity-indicator {:animating true}]]))
 
 (defn on-navigation-change [event browser]
-  (let [{:strs [url canGoBack canGoForward]} (js->clj event)]
-    (when (and (not (:dapp? browser)) (not= "about:blank" url))
-      (re-frame/dispatch [:update-browser (assoc browser :url url)]))
+  (let [{:strs [url canGoBack canGoForward loading]} (js->clj event)]
+    (when (not= "about:blank" url)
+      (let [browser2 (assoc browser :url url)]
```

Comment:
usually, we're using `'` for this, `browser' (assoc browser :url url)]`

Reply:
@flexsurfer I think we are moving away from using `'` , in this case you could use `browser-with-url` or `raw-browser` -> `browser`

Reply:
Got it, will fix!

Reply:
Done! Using `raw-browser -> browser`